cmd/cli/jobflow.go

```diff
@@ -0,0 +1,69 @@
+package main
```

Comment:
@googs1025 please add the doc under docs/design/command-line-enhancement

Reply:
+1

Reply:
create pr: https://github.com/volcano-sh/volcano/pull/3584